Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN pip install -r requirements/test_requirements.txt
 ADD . ${HOME}/mozilla-schema-generator
 ENV PATH $PATH:${HOME}/mozilla-schema-generator/bin
 
-RUN pip install -e ${HOME}/mozilla-schema-generator
+RUN pip install --no-dependencies -e ${HOME}/mozilla-schema-generator
```

Comment:
This is an unrelated efficiency boost. We already install dependencies based on requirements.txt before this line, so we can save some time by avoiding validating dependencies here.